cypress/integration/channel/edit_message_spec.js

```diff
@@ -109,4 +109,46 @@ describe('Edit Message', () => {
             });
         });
     });
+    it('M13542 Open edit modal immediately after making a post', () => {
+        // 1. Login as "user-1" and go to /
+        cy.login('user-1');
```

Comment:
I renamed this command to `apiLogin` just to be clear that it's calling the API directly.  Please update.

Reply:
Changed